frontend/packages/model-registry/manifests/rhoai/model-registry-ui-deployment.yaml

```diff
@@ -0,0 +1,6 @@
+- op: add
+  path: /spec/template/spec/containers/0/args
+  value: 
+    - "--standalone-mode=false"
+    - "--default-platform=false"
+    - "--port=8080"
```

Comment:
Why is this specific for RHOAI?

Reply:
nah, it was a test to see if I could get an overlay working, I'm gonna remove that folder and add it later if needed.